src/number_theory/modular.lean

```diff
@@ -342,6 +410,53 @@ begin
   field_simp [norm_sq_denom_ne_zero, norm_sq_ne_zero, S]
 end
 
+/-- The standard (closed) fundamental domain of the action of `SL(2,ℤ)` on `ℍ`. -/
+def fundamental_domain : set ℍ :=
+{z | 1 ≤ (z : ℂ).norm_sq ∧ |z.re| ≤ (1 : ℝ) / 2}
+
+/-- The standard open fundamental domain of the action of `SL(2,ℤ)` on `ℍ`. -/
+def fundamental_domain_open : set ℍ :=
+{z | 1 < (z : ℂ).norm_sq ∧ |z.re| < (1 : ℝ) / 2}
+
+localized "notation `𝒟` := modular_group.fundamental_domain" in modular
+
+localized "notation `𝒟ᵒ` := modular_group.fundamental_domain_open" in modular
+
+lemma abs_two_mul_re_lt_one_of_mem_fdo (h : z ∈ 𝒟ᵒ) : |2 * z.re| < 1 :=
```

Comment:
If we're going to call this `fdo` in lemma names, then perhaps we should call the `def` `fdo` too to match.

Reply:
I'm on the fence here but so I'll do as suggest for consistency.